starter-code/apple-pie-starter-code/main.css

```diff
@@ -0,0 +1,41 @@
+
+/*Some necessary adjustments*/
+h1, body, html, main {
+  margin: 0;
+}
+/*Import the font and set the default size*/
+html {
+  font-family: 'Open Sans', sans-serif;
+  font-size: 18px
+}
+.crop{ 
+  background-color: green;
+  margin: auto;
+  width: 888px;
+  height: 480px;
+  overflow: hidden;
+}
+
+.crop img{
+  position: relative; 
+  width: 888px;
+  height: 788px;
+  margin: -100px 0 0 0;
+}
+
+h1{
+    color: white;
+    position: absolute; 
```

Comment:
There are many ways to center elements, but usually you are going to use 
```
text-align: center; /* (inline elements) */
```
or 
```
margin: 0 auto; /* (block element) */ 
```
Here you have an amazing guide: https://css-tricks.com/centering-css-complete-guide/